chrome/map.js

```diff
@@ -476,7 +476,7 @@ SectorMap.prototype = {
 		}
 		this.drawSavedPath(this.get2DContext());
 		this.markShipTile(this.get2DContext());
-		this.distanceDiv.innerHTML = "Distance to " + this.sector.sector + " [" + loc.x + ", " + loc.y + "]: " + apsSpent + " APs<br>&nbsp;"; //innerHTML to accomodate infinity symbol
+		this.distanceDiv.innerHTML = "Distance to " + this.sector.sector + " [" + loc.x + ", " + loc.y + "]:<br>" + apsSpent + " APs<br>&nbsp;"; //innerHTML to accomodate infinity symbol
```

Comment:
This is part of a fix to the problem of the #rightSideBar moving when you hover over the minimap in sectors with long names.

Reply:
good catch